sr/comp/mixtape/cli.py

```diff
@@ -104,28 +128,43 @@ def play(args):
     scheduler.run()
 
 
-def verify_tracks(mixtape_dir, tracks):
+def verify_track(mixtape_dir, filename):
+    path = os.path.join(mixtape_dir, filename)
+    if not os.path.exists(path):
+        print(path, "doesn't exist!")
+
+
+def verify_tracks(mixtape_dir, tracks, matches):
     for track in tracks:
         try:
             filename = track['filename']
         except KeyError:
             try:
                 filename = track['obs_video']
+                if '{match_num' in filename:
```

Comment:
Should we not be looking for the whole thing here?

```suggestion
                if '{match_num}' in filename:
```